addon/routes/building/edit/index.js

```diff
@@ -1,3 +1,13 @@
 import Route from "@ember/routing/route";
+import { inject as service } from "@ember/service";
 
-export default class BuildingEditIndexRoute extends Route {}
+export default class BuildingEditIndexRoute extends Route {
+  @service building;
+
+  model() {
+    // reset the newRecord
+    console.log("model buidling");
+    this.building.newRecord = null;
```

Comment:
Would be nice if this were in `beforeModel` since the model hooks is just for "fetching" model data